app/views/layouts/main.volt

```diff
@@ -4,19 +4,44 @@
     <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
 
     <head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# blog: http://ogp.me/ns/blog#">
+        <title>Phosphorum - Discussions</title>
```

Comment:
The below 3 meta tags **must** come first in the head; any other head content must come **after** these tags
```html
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
```

Also note, you have
```html
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
```
and
```html
<meta charset="utf-8" />
```

here